RecoEgamma/ElectronIdentification/python/Identification/cutBasedElectronID_Fall17_94X_V1_Preliminary_cff.py

```diff
@@ -0,0 +1,190 @@
+    absEInverseMinusPInverseCut    = 0.0278    , # absEInverseMinusPInverseCut
+    # conversion veto cut needs no parameters, so not mentioned
+    missingHitsCut                 = 1          # missingHitsCut
+    )
+
+WP_Tight_EE = EleWorkingPoint_V4(
+    idName                         = idName  , # idName
+    full5x5_sigmaIEtaIEtaCut       = 0.0305  , # full5x5_sigmaIEtaIEtaCut
+    dEtaInSeedCut                  = 0.00567 , # dEtaInSeedCut
+    dPhiInCut                      = 0.0165  , # dPhiInCut
+    hOverECut_C0                   = 0.026   , # hOverECut
+    hOverECut_CE                   = 0.5     ,
+    hOverECut_Cr                   = 0.201   ,
+    relCombIsolationWithEALowPtCut = 0.094  , # relCombIsolationWithEALowPtCut
+    relCombIsolationWithEAHighPtCut= 0.094  , # relCombIsolationWithEAHighPtCut
+    absEInverseMinusPInverseCut    = 0.0158 , # absEInverseMinusPInverseCut
+    # conversion veto cut needs no parameters, so not mentioned
+    missingHitsCut                 = 1          # missingHitsCut
+    )
+
+# Second, define what effective areas to use for pile-up correction
+isoInputs = IsolationCutInputs_V2(
+    # phoIsolationEffAreas
+    "RecoEgamma/ElectronIdentification/data/Fall17/effAreaElectrons_cone03_pfNeuHadronsAndPhotons_92X.txt"
+)
+
+
+#
+# Set up VID configuration for all cuts and working points
+#
+
+cutBasedElectronID_Fall17_94X_V1_Preliminary_veto = configureVIDCutBasedEleID_V4(WP_Veto_EB, WP_Veto_EE, isoInputs)
+cutBasedElectronID_Fall17_94X_V1_Preliminary_loose = configureVIDCutBasedEleID_V4(WP_Loose_EB, WP_Loose_EE, isoInputs)
+cutBasedElectronID_Fall17_94X_V1_Preliminary_medium = configureVIDCutBasedEleID_V4(WP_Medium_EB, WP_Medium_EE, isoInputs)
+cutBasedElectronID_Fall17_94X_V1_Preliminary_tight = configureVIDCutBasedEleID_V4(WP_Tight_EB, WP_Tight_EE, isoInputs)
+
+
+# The MD5 sum numbers below reflect the exact set of cut variables
+# and values above. If anything changes, one has to 
+# 1) comment out the lines below about the registry, 
+# 2) run "calculateMD5 <this file name> <one of the VID config names just above>
+# 3) update the MD5 sum strings below and uncomment the lines again.
+#
+
+#central_id_registry.register(cutBasedElectronID_Fall17_94X_V1_Preliminary_veto.idName,
+#                             '0025c1841da1ab64a08d703ded72409b')
+#central_id_registry.register(cutBasedElectronID_Fall17_94X_V1_Preliminary_loose.idName,
+#                             'c1c4c739f1ba0791d40168c123183475')
+#central_id_registry.register(cutBasedElectronID_Fall17_94X_V1_Preliminary_medium.idName,
+#                             '71b43f74a27d2fd3d27416afd22e8692')
+#central_id_registry.register(cutBasedElectronID_Fall17_94X_V1_Preliminary_tight.idName,
+#                             'ca2a9db2976d80ba2c13f9bfccdc32f2')
+
+
+### for now until we have a database...
+cutBasedElectronID_Fall17_94X_V1_Preliminary_veto.isPOGApproved = cms.untracked.bool(False)
```

Comment:
this seems counter-productive.
Why does the POG propose an ID to be in the release if it is not approved by the POG?

Reply:
Changes implemented here:
https://github.com/lsoffi/cmssw/blob/CMSSW_9_4_0_pre3_TnP/RecoEgamma/ElectronIdentification/python/Identification/cutBasedElectronID_Fall17_94X_V1_Preliminary_cff.py
These V1 IDs are approved and ready to be merged in CMSSW.